python_modules/libraries/dagster-docker/dagster_docker/utils.py

```diff
@@ -64,4 +64,4 @@ def parse_env_var(env_var_str: str) -> Tuple[str, str]:
         split = env_var_str.split("=")
         return (split[0], split[1])
     else:
-        return (env_var_str, os.getenv(env_var_str))
+        return (env_var_str, os.environ[env_var_str])
```

Comment:
did you peak at callers of this to verify? some concern this could break people if the typehint was wrong and we sometimes have None here

Reply:
In all calling cases, the result is being forwarded to `docker.client.containers.create`, which has this in its docs:

>Environment variables to set inside the container, as a dictionary or a list of strings in the format ["SOMEVARIABLE=xxx"].

I'm not 100% certain but I'm pretty sure this doesn't take `None`. But, given that if Nones _were_ being passed before, it wasn't breaking anything, I reverted the change here.